.github/workflows/update-flathub.yml

```diff
@@ -0,0 +1,43 @@
+name: update-flathub
+on:
+  workflow_dispatch:
```

Comment:
This is fine for now of course. For the future, we'd move this as a job into release.yml and let that run after the `cockpituous` job?

Reply:
Correct.  That's the plan.  But I'd like to do this manually for the next couple of releases to be honest.  My next step (for next release) would be to have the PR automatically opened from the workflow.